specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2018-02-01-preview/containerInstance.json

```diff
@@ -1030,6 +1120,73 @@
         }
       }
     },
+    "AsyncOperation": {
```

Comment:
This is not the way we model long running operation at all. Please see for example:
https://github.com/Azure/azure-rest-api-specs/blob/8b34c73ead836efa86806e7da555ade6692cbde4/specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json#L403